plugins/ps/subcommands/restore

```diff
@@ -7,11 +7,20 @@ source "$PLUGIN_AVAILABLE_PATH/ps/functions"
 ps_restore_cmd() {
   declare desc="starts all apps with DOKKU_APP_RESTORE not set to 0 via command line"
   local cmd="ps:restore"
+
+  if which parallel > /dev/null 2>&1; then
+    dokku_apps | parallel dokku ps:start
+    return
```

Comment:
@josegonzalez I attached the code below in the PR #2555, it adds a hard dependency on parallel though.